File: materi/pertemuan_8/java/Helo.java
public class Helo {
    public static void main(String[] args) {
        hello();
        cetakNama("Nadia", "Bandung");

    }

    static void hello() {
        System.out.println("Helo world");
    }

    static void cetakNama(String nama, String kota) {
        System.out.println("Nama saya " + nama + ". Saya tinggal di kota " + kota);
    }
}

File: materi/pertemuan_8/js/helo.js
// helo();
//cetak_nama("Nadia");
cetak_nama("Nadia", "Bandung");

function nama_fungsi() {
    // block code yang dieksekusi
}

// function tanpa parameter
function helo() {
    console.log("Hello world");
}

function cetak_nama(nama, kota = "Sukabumi") {
    console.log("Nama saya " + nama + ". Saya tinggal di kota " + kota);
}